djangocms_navigation/utils.py

```diff
@@ -19,7 +57,9 @@ def supported_models():
 
 @lru_cache(maxsize=1)
 def supported_content_type_pks():
-    app_config = apps.get_app_config("djangocms_navigation")
+    MenuContent = get_model('MENU_MODEL')
+    app_label = MenuContent._meta.app_label
+    app_config = apps.get_app_config(app_label)
```

Comment:
I'm thinking that because this is repeated this block should be a utility to get the apps config, an internal `get_app_config_from_model()` Returns app_config